db/dynamodb-global-table.yml

```diff
@@ -50,6 +52,10 @@ Metadata:
         default: Primary key field
```

Comment:
Could be changed to "Partition key field"

Reply:
done